homeassistant/components/sensor/darksky.py

```diff
@@ -97,6 +97,346 @@
                              ['currently', 'hourly', 'daily']],
 }
 
+CONDITION_PICTURES = {
+    'clear-day': ('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZ'
```

Comment:
Adding them here will mean they end up in the state machine, in the database and slow down initial load of the app. Not allowed.

Please remove all the inline images and instead add them to a new folder in the static folder here: https://github.com/home-assistant/home-assistant/tree/dev/homeassistant/components/frontend/www_static/images .